pyleadsheet/main.py

```diff
@@ -81,3 +82,6 @@ def main():
 
     elif args['generate']:
         return generate(args)
+
+if __name__ == '__main__':
+    main()
```

Comment:
OK, I'll get rid of it. Looks like I need to learn a little more about how setup.py and virtualenv play together, I was not aware of this.